modules/mobile_session.lua

```diff
@@ -101,8 +101,15 @@ function mt.__index:ExpectNotification(funcName, ...)
     data.sessionId == self.sessionId
   end
   args = table.pack(...)
-  if #args[1] > 0 or args[1].n == 0 then
-    args = args[1]
+
+  if #args ~= 0 then
```

Comment:
@pestOO I think it's unneeded, because it's a common case of expecting notification without any valuable parameters (e. g. `OnAudioPassThru` or `OnHashChange`) from SDL in **defined** (not default) session.